internal/cmdliveinit/cmdliveinit.go

```diff
@@ -80,9 +81,10 @@ func NewRunner(ctx context.Context, factory k8scmdutil.Factory,
 	r.Command = cmd
 
 	cmd.Flags().StringVar(&r.Name, "name", "", "Inventory object name")
-	cmd.Flags().BoolVar(&r.Force, "force", false, "Set inventory values even if already set in Kptfile")
+	cmd.Flags().BoolVar(&r.Force, "force", false, "Set inventory values even if already set in Kptfile or ResourceGroup file")
 	cmd.Flags().BoolVar(&r.Quiet, "quiet", false, "If true, do not print output message for initialization")
 	cmd.Flags().StringVar(&r.InventoryID, "inventory-id", "", "Inventory id for the package")
+	cmd.Flags().StringVar(&r.RGFileName, "rg-filename", rgfilev1alpha1.RGFileName, "Name of the file holding the ResourceGroup resource.")
```

Comment:
nit: Shall we name this to `rg-file` to align with other flag names like `log-file`?

Reply:
Fixed the name. I'm not sure if we need to point out the format of the file since the file is being generated by kpt. The user doesn't have to create the file themselves.